alternator/executor.cc

```diff
@@ -3573,6 +3573,11 @@ static rjson::value encode_paging_state(const schema& schema, const service::pag
             ++exploded_ck_it;
         }
     }
+    //FIXME conflicts with column names
+    rjson::add(last_evaluated_key, "region", rjson::empty_object());
+    rjson::add(last_evaluated_key["region"], "S", rjson::from_string(to_string(pos.region())));
+    rjson::add(last_evaluated_key, "weight", rjson::empty_object());
+    rjson::add(last_evaluated_key["weight"], "N", static_cast<int>(pos.get_bound_weight()));
```

Comment:
Might be solved by using a prefix that is an illegal attribute name (alternator terminology for columns).